README.md

```diff
@@ -1,8 +1,37 @@
-# epuck_ros2
+# E-Puck Driver for ROS2
 [![Build Status](https://travis-ci.com/cyberbotics/epuck_ros2.svg?branch=master)](https://travis-ci.com/cyberbotics/epuck_ros2)
 [![license - apache 2.0](https://img.shields.io/:license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Version](https://img.shields.io/github/v/tag/cyberbotics/epuck_ros2?label=version)](http://wiki.ros.org/epuck_ros2)
 
+<img src="./assets/pi-puck.jpg" width="400px" />
+
+This ROS2 package adds support for the [e-puck](https://www.gctronic.com/doc/index.php/e-puck2) physical robot with [Pi-puck extension](https://www.gctronic.com/doc/index.php?title=Pi-puck).
+Please use the following links for the instructions:
+- [Installation](../installation/README.md)
+- [Getting Started](#getting-started)
+- [Examples](https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2_epuck/EPUCK_ROS2.md)
+- [Simulation](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_epuck)
+
+## Getting Started
+Make sure you followed the [installation tutorial](./installation), so you have ROS2 and `epuck_ros2` installed on your robot.
```

Comment:
```suggestion
Make sure you followed the [installation tutorial](../installation/README.md), so you have ROS2 and `epuck_ros2` installed on your robot.
```

Reply:
`../installation/...` didn't work, so I changed it to `./installation`

Reply:
Ok good I saw you changed the other (which was using `../installation`) too.